src/calculateRentalCost.js

```diff
@@ -26,9 +26,21 @@
  *
  * @return {number}
  */
+const longTerm = 7;
+const longTermDiscount = 50;
+const shortTerm = 3;
+const shortTermDiscount = 20;
```

Comment:
those should be uppercased - [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 2